### DISCUSSION TITLE
Integration with OwnTracks Recorder?

### DISCUSSION THREAD

**Discussion by GitPuffy** [MartinPham/FindMySync#18] [Ideas]
I haven’t installed your FindMySync application (yet) but am curious to know if it would integrate with OwnTracks Recorder.  I use OwnTracks to record my whereabouts from my iPhone and having the ability to record my AirTag devices into OwnTracks would be great also. 

OwnTracks has a MQTT and HTTP receiving mechanism.  I use the HTTP personally.  Can the output of FindMySync send to a HTTP URL?

https://github.com/owntracks/recorder
https://github.com/owntracks

Thank you!

**Comment by SimonBard** [comment]
It should be easy if you know how to handle xcode. I can see in the code it should be fairly easy, but I do not know how to compile the code. 

The http post looks like this: 
index.php/apps/phonetrack/logGet/490efcfcb9bb71lsfohiweiohgrigw/schwarz_flach_eckig?lat=80.934635199999995&lon=40.583498099999998&alt=0&acc=14.93343496336773&bat=1&sat=SAT&speed=SPD&bearing=DIR&timestamp=1712174836000
